packages/browser/src/Component/ItownsUtil.js

```diff
@@ -96,26 +43,20 @@ export function setup3DTilesLayer(layer, layerManager, itownsView) {
  * Setup and add 3D tiles to an itowns view
  *
  * @param {object} config3DTilesLayers An object containing 3D Tiles layers configs
- * @param {LayerManager} layerManager a layer manager
  * @param {itowns.View} itownsView - the itowns view
- * @returns {Map<itowns.C3DTilesLayer>} a map of each 3d tiles layer
  */
-export function add3DTilesLayers(
-  config3DTilesLayers,
-  layerManager,
-  itownsView
-) {
+export function add3DTilesLayers(config3DTilesLayers, itownsView) {
```

Comment:
As this method is not really that relevant,  don't you think that we can drop this method and do the loop on the layers directly in the code ?

Reply:
I agree even more we may want to customize the for loop creation by adding a style for example. In my opinion all addLayerFromConfig should be removed because the itowns API is enough.

If everyone agree I will remove these functions ?

Reply:
I'm fine with this.

I guess code examples  are always nice to have around, so go for it if you want